lib/spack/spack/test/cmd/spack_is_git_repo.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from __future__ import print_function
+
+import spack
+import pytest
+
+from llnl.util.filesystem import mkdirp
+
+from spack.util.executable import which
+
+git = which("git")
```

Comment:
Could you add a `skipif` check like the `git_fetch` testing module for when Git isn't available?

Reply:
The `skipIf` check is already implemented in line L17+:
```
pytestmark = pytest.mark.skipif(
    not git, reason="we need git to test if we are in a git repo"
)
```